src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -1730,6 +1730,16 @@ bool checkValid(MethodSymbol candidate, bool isCandidate, DiagnosticBag specific
                         }
                     }
                 }
+                else if (LanguageVersion >= MessageID.IDS_FeatureAsyncMain.RequiredVersion() && taskEntryPoints.Count == 1 && viableEntryPoints.Count == 1)
+                {
+                    var info = new CSDiagnosticInfo(
+                         ErrorCode.WRN_SyncAndAsyncEntryPoints,
+                         args: Array.Empty<object>(),
+                         symbols: ImmutableArray<Symbol>.Create(viableEntryPoints.First(), taskEntryPoints.First()),
```

Comment:
The CI says: 

> error CS0117: 'ImmutableArray<Symbol>' does not contain a definition for 'Create'

while I see the [API reference](https://docs.microsoft.com/en-us/dotnet/api/system.collections.immutable.immutablearray.create?view=netcore-3.1#System_Collections_Immutable_ImmutableArray_Create__1___0___0_) says the Create exists. I'm confused on what's the reason.

Reply:
I think `Create<T>` is on the static class `ImmutableArray`, not the struct `ImmutableArray<T>`.

Reply:
In other words just delete the `<Symbol>` bit and it should work.

Reply:
Thanks!